tests/root_chain/contracts/root_chain/test_root_chain.py

```diff
@@ -12,24 +12,39 @@ def root_chain(t, get_contract):
     return contract
 
 
-def testNextBlockNumber(t, root_chain):
-    assert 1000 == root_chain.childBlockInterval()
-    assert 1000 == root_chain.nextWeekOldChildBlock(0)
-    assert 1000 == root_chain.nextWeekOldChildBlock(55)
-    assert 2000 == root_chain.nextWeekOldChildBlock(1000)
-    assert 2000 == root_chain.nextWeekOldChildBlock(1001)
+def test_deposit(t, u, root_chain):
+    owner, value_1 = t.a0, 100
+    blknum = root_chain.getDepositBlock()
+    root_chain.deposit(value=value_1)
+    assert root_chain.getChildChain(blknum)[0] == get_merkle_of_leaves(16, [owner + b'\x00' * 31 + u.int_to_bytes(value_1)]).root
+    assert root_chain.getChildChain(blknum)[1] == t.chain.head_state.timestamp
+    assert root_chain.getDepositBlock() == blknum + 1
 
 
-def test_deposit(t, root_chain):
-    owner, value_1 = t.a1, 100
-    null_address = b'\x00' * 20
-    tx = Transaction(0, 0, 0, 0, 0, 0,
-                     owner, value_1, null_address, 0, 0)
-    tx_bytes = rlp.encode(tx, UnsignedTransaction)
+def test_start_deposit_exit(t, u, root_chain, assert_tx_failed):
+    owner, value_1 = t.a0, 100
     blknum = root_chain.getDepositBlock()
-    root_chain.deposit(tx_bytes, value=value_1)
-    assert root_chain.getChildChain(blknum)[0] == get_merkle_of_leaves(16, [tx.hash + tx.sig1 + tx.sig2]).root
-    assert root_chain.getChildChain(blknum)[1] == t.chain.head_state.timestamp
+    root_chain.deposit(value=value_1)
+    deposit_tx_hash = get_deposit_hash(owner, value_1)
+    expected_utxo_pos = blknum * 1000000000
+    expected_created_at = t.chain.head_state.timestamp
+    merkle = FixedMerkle(16, [deposit_tx_hash], True)
+    proof = merkle.create_membership_proof(deposit_tx_hash)
+    root_chain.startDepositExit(expected_utxo_pos, value_1, proof)
+    utxo_pos, created_at = root_chain.getNextExit()
+    assert utxo_pos == expected_utxo_pos
+    assert created_at == expected_created_at
+    assert root_chain.exits(utxo_pos) == ['0x82a978b3f5962a5b0957d9ee9eef472ee55b42f1', 100]
+    # Same deposit cannot be exited twice
+    assert_tx_failed(lambda: root_chain.startDepositExit(utxo_pos, value_1, proof))
```

Comment:
I think these should be broken up into individual tests. It's typically easier to read (and find errors) if we break everything up into ~1 assert per test but it's not the end of the world if it's >1.